{"name": "Неуспешная авторизация при запросе с неверным паролем", "status": "passed", "description": "Авторизация не проходит, введен неверный пароль", "steps": [{"name": "Отправка запроса на авторизацию пользователя", "status": "passed", "parameters": [{"name": "body", "value": "{'email': 'spreston@example.com', 'password': 'random_password'}"}], "start": 1717941760492, "stop": 1717941760900}], "start": 1717941760491, "stop": 1717941760903, "uuid": "be5f3543-239d-44da-a27b-6fae3a81ccd4", "historyId": "df0d8c73aeef8704d99a2105fe08b380", "testCaseId": "df0d8c73aeef8704d99a2105fe08b380", "fullName": "tests.test_auth_user.TestAuthUser#test_cannot_auth_user_with_invalid_password", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_auth_user"}, {"name": "subSuite", "value": "TestAuthUser"}, {"name": "host", "value": "MacBook-Pro-Ivan-4.local"}, {"name": "thread", "value": "47205-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_auth_user"}]}